src/tasks/installers/operator.ts

```diff
@@ -184,6 +186,29 @@ export class OperatorTasks {
           }
         }
       },
+      {
+        title: 'Create backup and restore CRDs',
+        task: async (ctx: any, task: any) => {
```

Comment:
if file does not exist I prefer `${task.title}...skipped.` like with other crd

Reply:
ok